src/Components/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,62 @@
+import './ProductDetail.css'
+import {useState , useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+import axios from 'axios';
+import {Loader} from '../index'
+import { useProducts } from '../../Context/products-context'
+
+function ProductDetail() {
+    const [product , setProduct] = useState({})
+    const [isLoading, setIsLoading] = useState(false)
+
+    const {id} = useParams();
+    console.log(id)
+    // const {products} = useProducts()
```

Comment:
remove commented code

Reply:
done removed all console log and commented code